### DISCUSSION TITLE
How do I delete headers when using responseInterceptor?

### DISCUSSION THREAD

**Discussion by thealtofwar** [chimurai/http-proxy-middleware#712] [Q&A] [answered]
I would like to delete a header from the response of the proxy
```js
onProxyRes: responseInterceptor(async (responseBuffer, proxyRes, req, res) => {
    const response = responseBuffer.toString('utf8'); 
    delete proxyRes.headers["Content-Security-Policy"];
    return response.replaceAll('//' + proxySite, '//' + hostName).replaceAll("//" + proxyCDN, "//" + hostCDN);
}
```
However the proxy still sends the CSP. 
```sh
~$ curl --head localhost:3000
HTTP/1.1 200 OK
some headers ommitted ...
content-security-policy: default-src 'self'; image-src: 'self' https://thirdpartycdn.com
```

**Comment by chimurai** [comment, marked as answer]
It's document here:
https://github.com/chimurai/http-proxy-middleware/blob/master/recipes/response-interceptor.md#manipulate-response-headers

Credits go to @stefanwright1988 for updating the docs (https://github.com/chimurai/http-proxy-middleware/pull/724)

**Comment by dcarias0388** [comment]
Y como pudiera eliminar un encabezado en el onProxyReq?....porque solo me deja modificarlo pero no eliminar.
